src/components/node/news_story/index.tsx

```diff
@@ -1,12 +1,29 @@
+/**
+ * ### Overview
+ * News Story represents an individual story within a Facility. These are used for human-interest articles.
+ *
+ * News Story expects nodes of type {@link NodeNewsStory}.
+ *
+ * ### View modes
+ * Full page: {@link NewsStoryFull}
```

Comment:
Currently unknown...it will be based on the output needs. Typically though there's not much call to have more than 4-5 displays of a given Drupal entity.